easybuild/easyconfigs/c/CP2K/fix_compile_date_lastcvs.patch

```diff
@@ -1,14 +1,11 @@
---- makefiles/Makefile.orig	2010-12-10 05:35:15.000000000 +0100
-+++ makefiles/Makefile	2011-01-24 10:14:35.000000000 +0100
-@@ -149,9 +149,9 @@
+--- makefiles/Makefile.orig	2012-08-21 14:35:17.000000000 +0300
++++ makefiles/Makefile	2013-11-12 12:41:29.361582641 +0200
+@@ -194,7 +194,7 @@
```

Comment:
I don't like this patches of an existing patch file, is it required @tloizou?

we need to be very careful here not to break existing CP2K builds...